### PR TITLE
resolve issue #192

### PR DESCRIPTION
**Pull request by weiwang-gsa**

### CODE REVIEW

File: pages/insights/index.njk [hunk 1]
```diff
@@ -20,10 +20,14 @@ hero:
 
     <div class="grid-container padding-top-205">
       <div class="grid-row grid-gap">
-      {% for insight in collections.insights | sort(attribute='sortorder') %}
+      {% for insight in collections.insights | sort(attribute='data.sortorder') %}
         <div class="tablet:grid-col-4 margin-y-1 padding-bottom-6">
           <div class="flex-column">
-          <a class="usa-link text-no-underline text-black" href="{{ insight.url | url }}" aria-label="link to {{ insight.data.title}}">
+             {% if insight.data.externallink %}
+               <a class="usa-link text-no-underline" href="{{ insight.data.externallink | url }}">
```
Comment (mark-meyer) [inline pages/insights/index.njk:27]: @weiwang-gsa Do we need to also include the `aria-label` here too for accessibility?

File: pages/insights/future-work-starts-today.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/insight
+title: The future of work starts today
+date: 2023-02-14
+author: Chuck Hardy
+position: Chief Architect
+externallink: https://www.gsa.gov/blog/2023/02/13/the-future-of-work-starts-now
+image: 
+  url: /assets/img/insights-future-work.jpg
+  alt: "Five individuals standing in the hallway of workplace innovation lab and doing ribbon cutting ceremonies. "
```
Comment (mark-meyer) [inline pages/insights/future-work-starts-today.md:10]: I would defer to @brentryanjohnson on this, but maybe something like "GSA leadership opening the Workplace Innovation Lab during a ribbon cutting ceremony." might be add a little more interesting info.

**Comment by weiwang-gsa** [comment]
@mark-meyer great catch and thank you so much, I've updated the code accordingly, also made adjustment for the page title to match external link title. Please review again when you get a chance.

**Review by mark-meyer — APPROVED** [review]
Looks good to me. Great idea with putting the external link in the front-matter — that works really well.

**Pull request merged**